### ISSUE TITLE
Visual Focus Indicator Lost when clicking on search button

### ISSUE THREAD

**Issue by davidstenger** [18F/web-design-standards#1652]
**Issue**: 
When Search button activated, the new search field is appeared on the page and  keyboard users are unable to track keyboard focus on the.  Having the visual focus is critical for a sighted keyboard user to track where they are on the page.  
See attachment [FocusLost_ExtendedHeader.png](https://drive.google.com/open?id=0ByQeU_iOqP_iTEFhNXYyczQwNUk)

**Solution**:
Ensure keyboard users can track their focus at all times. After activation of Searche button, set focus to the Search Small edit field.

**Comment by toolness** [comment]
Hmm, I'm having trouble reproducing this one. Looking at the [history of `search.js`](https://github.com/18F/web-design-standards/commits/develop/src/js/components/search.js), though, I'm also not seeing it explicitly fixed anywhere--it looks like the code has always *tried* to focus on the input field when it shows, and right now it appears to be succeeding at it.

I'm going to close this issue for now, but feel free to re-open it if you disagree!
